### DISCUSSION TITLE
On iOS the notifications list heading font size is too large

### DISCUSSION THREAD

**Discussion by manuelmeister** [community/community#51911] [Mobile] [answered]
### Select Topic Area

Product Feedback

### Body

Since I‘ve update the Github App, the font size of the notifications view is too large. I assume this has to do with the dynamic font sizes. But they are too large compared to other apps that use the dynamic font sizes as well.

<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.106.0 (94961686)
Device: iPhone 12 Pro (iOS 16.2)
</details>

**Comment by iskae** [comment, marked as answer]
Hello @manuelmeister 👋 

Thanks for reaching out ❤️ 

Unfortunately I was not able to reproduce the issue on our side 😅  Would you be able to provide a screenshot for us so that we can understand the issue better?

Thanks in advance!

**Reply by manuelmeister** [reply]
Strangley it has resolved itself. Thank you!
